crates/dreamchecker/src/lib.rs

```diff
@@ -845,6 +851,109 @@ impl<'o> AnalyzeObjectTree<'o> {
         }
     }
 
+    /// Check that a ProcRef doesn't have fewer arguments than its parent
+    pub fn check_arg_count(&mut self, proc: ProcRef<'o>) {
+        if proc.is_builtin()
+        {
+            return;
+        }
+        if let Some(parent) = proc.parent_proc()
+        {
+            if parent.is_varargs() /* remove this later, it's a hack for objtree proc parents including root */ || parent.ty().get().is_root() /* seriously remove it */
```

Comment:
line length
```suggestion
			/* remove this later, it's a hack for objtree proc parents including root */
            if parent.is_varargs() || parent.ty().get().is_root() 
            /* seriously remove it */
```

Reply:
Only the is_root needs to be removed when #334 is solved. The varargs check is fine.

Reply:
```suggestion
            if parent.is_varargs() || 
            /* remove this later, it's a hack for objtree proc parents including root */
            parent.ty().get().is_root()
            /* seriously remove it */
```
this then, or just say that - it's wrong to have incredibly long lines with functionality hidden